public/i18n/en/translation.json

```diff
@@ -287,6 +290,25 @@
       "reboot_now": "Reboot now",
       "reboot_in_progress": "The unit is rebooting, please wait..."
     },
+    "storage": {
+      "title": "Storage",
+      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
```

Comment:
```suggestion
      "description": "You can keep a copy of your system logs on an external storage device, such as a USB stick. This can be useful for troubleshooting problems or for keeping a record of your system activity.",
```